apps/web/src/pages/activities/components/ActivityItem.tsx

```diff
@@ -164,4 +174,9 @@ const ListItem = styled.div<{ dark: boolean }>`
   margin-bottom: 15px;
   border-radius: 7px;
   color: ${({ dark }) => (dark ? colors.B80 : colors.B40)};
+
+  &:hover {
+    background-color: ${({ dark }) => (dark ? colors.B20 : colors.BGLight)};
+    border-radius: 7px;
```

Comment:
the border-radius is defined above, so it's not needed to be defined on hover

Reply:
Will merge it for now, since I want to finalize the release. Could fix it later 🙏